### DISCUSSION TITLE
Recognize special text to structure when save molfile, for example "H2O" or "Boc"

### DISCUSSION THREAD

**Discussion by ly-sniper** [epam/ketcher#466] [Ideas]
if I write some special text into editor by Text Tool, for example "H2O" or "Boc",
I hope ketcher can recognize them to real structure but not is simple text.

Is that possible to add a function to realize it?

**Comment by AndreiMazol** [comment]
Dear @ForBurning ,

Thank you for your interest to Ketcher.

Text tool has been created to add text labels with any comments. The functionality to recognize texts should be implemented as a part of 'Open Structure' dialog. We will discuss your idea internally.

Best regards,
Ketcher team

**Reply by ly-sniper** [reply]
Thank you for your reply @AndreiMazol.

If some abbr structure can be recognized by ketcher，we will save a lot of time to draw a Molfile or RXN.
The similar functions disappear on other editor like ChemDraw, Text tool is more convenient than 'Open Structure' dialog in this case.
